src/operator/nn/fully_connected.cc

```diff
@@ -331,5 +346,32 @@ NNVM_REGISTER_OP(_backward_FullyConnected)
 #endif
 .set_attr<FCompute>("FCompute<cpu>", FullyConnectedGradCompute<cpu>);
 
+
+// Higher order gradient for fully connected
```

Comment:
This is only second order as it cannot support higher order without using computation graph.

Reply:
right, my bad